std/algorithm/iteration.d

```diff
@@ -3621,14 +3620,32 @@ if (isInputRange!RoR && isInputRange!(ElementType!RoR))
                 _currentBack = typeof(_currentBack).init;
         }
 
+        void replaceCurrent(typeof(_current) current) @trusted
+        {
+            import std.algorithm.mutation : move;
```

Comment:
```suggestion
            import core.lifetime : move;
```
We really should alias and deprecate that stuff in `std.algorithm`. I think there was an attempt, what happened to it?..

Reply:
To start with it'd probably be good if there was a mention in the docs at https://dlang.org/phobos/std_algorithm_mutation.html#move